### PR TITLE
Added Count Maximum Possible Subsequence in CPP

### LINKED ISSUE #4459: Count Maximum Possible Subsequence
## 💥 Proposal

(A clear and concise description of what the proposal is.)

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

YES.

### PR DESCRIPTION
**Pull request by RaiyanMahin**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES.

### Description

Given two words and a string.
By using two words , Count How many different subsequence's can be created through the string.
Two subsequence's are considered to be different if they start or end
at different positions in the string.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes #4459

### CODE REVIEW

File: C-Plus-Plus/other/max_possible_subsequence.cpp [hunk 1]
```diff
@@ -0,0 +1,66 @@
+/*
+Given two words and a string.
+By using two words , Count How many different subsequence's can be created through the string.
+Two subsequence's are considered to be different if they start or end
+at different positions in the string.
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/other/max_possible_subsequence.cpp:10]: Add one line description about function above it.

File: C-Plus-Plus/other/max_possible_subsequence.cpp [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            /* here we will check if string s2 is second word or not
+            if yes we will increment the count_max_possible_subsequence
+            with current count1 value
+            */
+            count_max_possible_subsequence += count1;
+        }
+    }
+    return count_max_possible_subsequence;
+}
+int main()
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/other/max_possible_subsequence.cpp:40]: Add a new line above main() and also rewrite it like that,

Capture

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by RaiyanMahin** [comment]
@ankitaggarwal23 kindly review this also :D

File: C-Plus-Plus/other/max_possible_subsequence.cpp (hunk 1 above)
Comment (RaiyanMahin) [reply C-Plus-Plus/other/max_possible_subsequence.cpp:10]: i have already written it :/

File: C-Plus-Plus/other/max_possible_subsequence.cpp (hunk 2 above)
Comment (RaiyanMahin) [reply C-Plus-Plus/other/max_possible_subsequence.cpp:40]: done.

File: C-Plus-Plus/other/max_possible_subsequence.cpp [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+int main()
+{
+	string text, word1, word2;
+	
+        cout << "Enter the two words : \n";
+	cin >> word1 >> word2;
+	cout << "Enter the string : \n";
+	cin >> text;
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/other/max_possible_subsequence.cpp]: Align it properly.

File: C-Plus-Plus/other/max_possible_subsequence.cpp [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+By using two words , Count How many different subsequence's can be created through the string.
+Two subsequence's are considered to be different if they start or end
+at different positions in the string.
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+
+/* From this get_count_max_possible_subsequence() function we will count max different subsequence's can be created */
+
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/other/max_possible_subsequence.cpp]: Remove the space and use // instead of multi line comments

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by RaiyanMahin** [comment]
Done.

**Review by ankitaggarwal23 — APPROVED** [review]

**Comment by HarshCasper** [comment]
Please make an issue here and raise a PR, for an article as part of our Docs repository: https://github.com/TesseractCoding/NeoAlgo-Docs

It would be much appreciated. Thanks!

**Pull request merged**